src/announce.aggregatetransactions.command.ts

```diff
@@ -42,6 +89,11 @@ export abstract class AnnounceAggregateTransactionsCommand extends ProfileComman
                                            url: string) {
         const transactionHttp = new TransactionHttp(url);
         const listener = new Listener(url);
+        console.log(new TransactionFieldsTable(signedHashLockTransaction, signedAggregateTransaction, url).toString());
+        const shouldAnnounceTransaction = readlineSync.keyInYN('Do you want to announce this transaction? ');
+        if (!shouldAnnounceTransaction) {
```

Comment:
To match the other `anounce.transactions.command`, enclose the transaction announcement inside the if statement instead of using the negation form (looks cleaner).

E.g.:

`if (shouldAnnounceTransaction) { ... }`